tck/methods/sdk.ts

```diff
@@ -0,0 +1,35 @@
+import { Client, AccountId } from "@hashgraph/sdk";
+
+import { sdk } from "../sdk_data";
+import { SdkResponse } from "../response/sdk";
+import { SdkSetupParams } from "../params/sdk";
+
+export default {
+    setup: ({
+        operatorAccountId,
+        operatorPrivateKey,
+        nodeIp,
+        nodeAccountId,
+        mirrorNetworkIp,
+    }: SdkSetupParams): SdkResponse => {
+        let clientType: string;
+        if (nodeIp && nodeAccountId && mirrorNetworkIp) {
+            const node = { [nodeIp]: AccountId.fromString(nodeAccountId) };
+            sdk.client = Client.forNetwork(node);
+            clientType = "custom";
+        } else {
+            sdk.client = Client.forTestnet();
+            clientType = "testnet";
+        }
+        sdk.getClient().setOperator(operatorAccountId, operatorPrivateKey);
+        sdk.getClient().setRequestTimeout(30000);
```

Comment:
is there a reason for these `sdk.getClient` methods. Why not just use a `const client = ...` and then read it from the constant. This way you can have an immutable client.It's like a weird mix between the singleton pattern and a class. I might be missing something so you can correct me.

Reply:
Good catch—thanks for pointing this out. The current setup was designed with flexibility in mind, but it seems that using an immutable client behaves as expected. I’ll go ahead and adopt the immutable approach for simplicity. If we find that we need more flexibility in the future, we can revisit and adjust the SDK setup accordingly.